test/unit/spec/nbextensions/bulkImportCell/bulkImportCell-spec.js

```diff
@@ -53,6 +53,10 @@ define([
             }
             expect(cell.metadata.kbase.type).toBe('app-bulk-import');
             expect(cell.metadata.kbase.bulkImportCell).toBeDefined();
+            expect(cell.metadata.kbase.bulkImportCell.state).toEqual({
+                state: 'editingIncomplete',
+                selectedTab: 'configure'
+            });
```

Comment:
will there be additional tests where you move through different cell states?

Reply:
Yes, once there's a way to toggle through the different states. This is really just to get a few different states in place, and fix a startup bug.